### ISSUE TITLE
[Bug]: Declining Minetogether chat, Crashes when pressing / key

### ISSUE THREAD

**Issue by barriedHD** [FTBTeam/FTB-Modpack-Issues#1603]
### Modpack

Stoneblock 3

### Modpack version

Latest

### Describe the bug

I opened chat and it prompt me with minetogether chat, i declined and i got a message saying that i cant send messages whilst offline.  not even commands, so i knew you can bypass the chat by pressing / key (slash key). When i pressed it  chat opened up with the slash cursor in the chatbox and then minecraft became unresponsive and crashed.

### Steps to reproduce

1) decline minetogether chat.
2) press slash key

### Expected behaviour

unresponsive client followed by a crash

**Comment by barriedHD** [comment]
removed minetogether mod and fixed.

**Comment by OneComputerGuy** [comment]
Adding [1604](https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1604) for visibility since it's related

**Comment by Raidobw2** [comment]
Update to 1.0.2 and let me know if that is fixed, it should have been fixed in that version

**Comment by tjd46** [comment]
I just had this happen to me and I am on 1.0.2 here is a log if that helps https://pastebin.com/UtufkfB3

**Comment by OneComputerGuy** [comment]
@Raidobw2 It doesn't crash anymore while clicking or closing the chat, however, normal messages (like the ones the server leaves or the coordinates of bees spawning) are no longer visible either in the MineTogether tab (with MTogh disabled) or the commands tab.

These messages where sent as seen in the log but they don't appear when MineTogether is disabled (not sure if they appear when it's enabled):

```
[05Nov2022 20:37:18.383] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [CHAT] [Server]: getting a cup of tea, don't break the game!
[05Nov2022 20:41:36.676] [Render thread/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 1149 advancements
[05Nov2022 20:53:59.369] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [CHAT] Loot Bee has spawned at X: 3, Y: -5, Z: 16\nHurry up and get the loot before it despawns!
[05Nov2022 20:55:30.756] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [CHAT] Unknown or incomplete command, see below for error
[05Nov2022 20:55:30.756] [Render thread/INFO] [net.minecraft.client.gui.components.ChatComponent/]: [CHAT] /gamemode<--[HERE]
```

Another bug related to MineTogether comes when going to the pause menu (Esc key) > Green chat bubble icon at the top right > Try to toggle "Chat" from Enabled to Disabled, the mage shuts itself down (shows a "Saving world" screen and then closes the instance)

In the log you can see that the previous chat messages were posted, then I tried to disable MT and it closes the instance, not sure if that's the intended behavior from MT: https://pastebin.com/DAUwbn2G

**Comment by Raidobw2** [comment]
Thanks a lot, with both your feedback I have been able to relay more information to the MineTogether mod, which can be seen just above! 😄
For now if crashes still happen frequently the workaround will be to remove the MineTogether mod from any crashing/problematic games or servers

**Comment by Sunekaer** [comment]
Should be fixed in 1.1.0
